src/components/Button/Button.tsx

```diff
@@ -44,6 +44,8 @@ export interface Props {
   submit?: boolean;
   /** Renders a button that looks like a link */
   plain?: boolean;
+  /** Renders a button that looks like a link. Similar to `plain` buttons except the text is underlined and the text color and icon fill are inherited from the parent element */
+  monochrome?: boolean;
```

Comment:
> Downside is we’d need to deprecate a prop (outline on Button), and it would be a larger change.

We can definitely deprecate it. However do we foresee a case where we would want a button that looks like an `outline` button but without inheriting the text colour?

Reply:
I'm good with either approach. I like your proposed approach because it's clear what `monochrome` is doing. But at t the same time it could be confusing to add `monochrome` on a `primary` button and not see a change.

Having `monochrome` as its own variant does isolate it from the rest and make its more predictable, however the fact it looks `plain` might not be clear.

 ⚖️